servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -104,6 +106,134 @@
 
   /** The service that tries to check inodefiles with ttl set */
   private Future<?> mTTLCheckerService;
+  /** The configuration of {@link Constants#MASTER_TTLCHECKER_INTERVAL_MS} */
+  private int mTTLCheckerIntervalMs;
+
+  /**
+   * A bucket with all files whose ttl value lies in the bucket's time interval. The bucket's time
+   * interval starts at a specific time and lasts for {@link #mTTLCheckerIntervalMs}.
+   *
+   * Not thread-safe. Only for use related to {@link TTLBucketList}.
+   */
+  private class TTLBucket {
```

Comment:
The TTL stuff seems like an isolated class, perhaps move it to its own file to make `FileSystemMaster` less crowded?